### DISCUSSION TITLE
What the fuck is going on with versioning right now?

### DISCUSSION THREAD

**Discussion by Alfagun74** [ppetzold/nestjs-paginate#558] [General]
2 Weeks ago i was on major release 4.X.

Now, within one week the package skyrocketed to version 8:
![image](https://user-images.githubusercontent.com/22225051/227020191-e6aaea5c-16a7-47d3-aa77-39669dfd324d.png)

What is going on?

**Comment by ppetzold** [comment]
semver ?! 🤔

**Reply by Alfagun74** [reply]
I am familiar with the concept of semantic versioning, but it doesn't explain why you release so many breaking changes in such a spread out and quick succession of releases, instead of launching a beta version for the next major or just collecting a few commits before you release the next major release. It took 3 years to move from 1.0 to 4.0 but 8 days to move from 4.14 to 8.0. Feels very rushed and odd.

**Reply by ppetzold** [reply]
it's a simple helper method. some crazy release cycle would be overkill. you are free to choose any version and fork off if you are not liking the latest release 🤷‍♂️ 😄
